src/components/Settings/SaveSearchItem.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+	<main class>
+		<div class="item">
+			<span v-if="savedSearch.name">
+				{{ savedSearch.name }}
+			</span>
+			<span>
+				<kv-checkbox
+					v-model="showAlerts"
+				>
+					Email me when new loans become available
+				</kv-checkbox>
+			</span>
+			<span>
+				<kv-button
+					variant="secondary"
+				>
+					Delete Search
+				</kv-button>
+				<kv-button
+					variant="secondary"
+				>
+					View Loans
+				</kv-button>
+			</span>
+		</div>
+	</main>
+</template>
+
+<script>
+import KvButton from '~/@kiva/kv-components/vue/KvButton';
+import KvCheckbox from '~/@kiva/kv-components/vue/KvCheckbox';
+
+export default {
+	name: 'SaveSearchItem',
+	data() {
+		return {
+			showAlerts: false,
+
+		};
+	},
+	inject: ['apollo', 'cookieStore'],
```

Comment:
At the moment, this component doesn't need these.

Reply:
We'll eventually need these for queries around the loan count...so I'm ok with keeping them as that work is up soon.